plugins/services/src/js/pages/task-details/__tests__/TaskFileViewer-test.js

```diff
@@ -9,7 +9,7 @@ import TaskFileViewer from "../TaskFileViewer";
 
 let thisContainer, thisInstance;
 
-describe("TaskFileViewer", function() {
+describe.skip("TaskFileViewer", function() {
```

Comment:
I think we should expend 1hr here trying to have it not skippable.

Reply:
Did you get to do this yet?

Reply:
Not yet, but will have some time later today. A Will do it or unblock it.